sdk/python/tests/example_feature_repo_3.py

```diff
@@ -0,0 +1,20 @@
+from datetime import timedelta
```

Comment:
Can we rename `example_feature_repo_3.py` to something more descriptive like `feature_repo_with_missing_bq_source.py`?

Reply:
Yes definitely, thanks for the feedback.